unit-tests/live/serializable-device/test-l500-json-load.py

```diff
@@ -1,7 +1,7 @@
 # License: Apache 2.0. See LICENSE file in root directory.
 # Copyright(c) 2020 Intel Corporation. All Rights Reserved.
 
-#test:device L500*
+# test:device L500*
```

Comment:
This file, for example, has nothing but formatting changes, so I view it as cluttering the code review.
And, just in general, I prefer the spaces inside parentheses. FYI. :)

Reply:
reverted to original file